index.bs

```diff
@@ -1511,8 +1517,8 @@ these steps:
      Note: This lets a website immediately perform operations on the returned handles that
      might require user activation, such as requesting more permissions.
 
-     Issue(89): Rather than requiring the website to prompt separately for a writable directory,
-     we should provide some kind of API to request a writable directory in one step.
+     If |options|.{{OpenDirectoryPickerOptions/requestWritable}} is true, the
```

Comment:
This should probably be a separate step, rather than be part of the same step that runs the activation notification steps.

We could probably also make this more formal by having a step that invokes "requesting file system permission" explicitly, rather than having this more informal language (which would also let us specify what should happen if the user rejects that permission request).

Reply:
I've sketched this out in the most recent patch, though I assume we might also want to consider doing this for showOpenFilePicker and showSaveFilePicker?